php/ad-servers/class-ad-layers-ad-server.php

```diff
@@ -70,13 +70,24 @@ class Ad_Layers_Ad_Server extends Ad_Layers_Singleton {
 		public $handle = 'ad-layers';
 
 		/**
+		 * Capability required to manage the ad server settings. Defaults to
+		 * `manage_options`.
+		 *
+		 * @var string
+		 */
+		public $settings_capability = 'manage_options';
```

Comment:
I thought more about it and I'm going to switch all this code to leverage filters.